bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -2859,7 +2859,11 @@ public void internal_dispose_GC (long hDC, GCData data) {
 	OS.ReleaseDC (0, hDC);
 }
 
-boolean isXMouseActive () {
+/**
+ * Invokes platform specific functionality
+ * @noreference This method is not intended to be referenced by clients.
+ */
+protected boolean isXMouseActive () {
```

Comment:
You can not add such API as it is not existing on Mac/Linux, lacks documentation what is it about at all and probably doesn't make any sense there at all.

Reply:
doesn't @noreference prevent it from being API?

Reply:
Java API has to be the same for all ports, @noreference is just a documentation from Java POV which happen to have special treatment by PDE.